src/redux/books/booksSlice.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-param-reassign */
 import { createSlice } from '@reduxjs/toolkit';
-import { addBook, getBookList, removeBook } from '../api/getApiData';
+import { addBook, getBookList, removeBook } from '../../components/api/getApiData';
 
```

Comment:
- [ ] Great job so far in configuring the Redux store and handling `add` and `remove` book actions. Your work demonstrates a good understanding of Redux fundamentals. However, I noticed a slight deviation from the project requirements.

   - In this specific milestone, the project guidelines call for the creation of a `reducer` within your `slice` that handles the `add` and `remove` book actions. To align with the requirements, your code structure should resemble the following example:

```javascript
export const booksSlice = createSlice({
  name: "books",
  initialState,
  reducers: {
    addBook: (state, action) => {
      // Define your add reducer functionalities here
    },
    removeBook: (state, action) => {
      // Define your remove reducer functionalities here
    },
  },
})
```

It's important to note that making `API` calls isn't part of the requirements for this milestone. Please review your codebase and remove any code related to `API` calls as it's not within the scope of this task. Additionally, consider removing any `extraReducers` from your `slice` as they are not needed for this milestone.

Great work overall! 🚀 😊

Reply:
- [ ] Good job implementing the changes requested by the previous code reviewer, :clap: However, as he mentioned above, there is a little deviation from the project requirement, In this specific milestone, the project guidelines call for the creation of a reducer within your slice that handles the add and remove book actions. To align with the requirements, your code structure should resemble the following example:

`export const booksSlice = createSlice({`
 ` name: "books",`
  `initialState,`
 ` reducers: {`
   ` addBook: (state, action) => {`
   `   // Define your add reducer functionalities here`
 `   },`
   ` removeBook: (state, action) => {`
    `  // Define your remove reducer functionalities here`
  `  },`
`  },`
`})`

- [ ] It's important to note that making API calls isn't part of the requirements for this milestone. Please review your codebase and remove any code related to API calls as it's not within the scope of this task. Additionally, consider removing any extraReducers from your slice as they are not needed for this milestone. :slightly_smiling_face: